### PR TITLE
Per-Placeholder Default Values

### PR DESCRIPTION
**Pull request by Oribuin**
This adds the ability for people to customize the default values for each individual placeholders instead of a global one.

This will use placeholders.yml

### CODE REVIEW

**Review by PeachesMLG — APPROVED** [review]
Looks good to me, but im dum so @dlsf ?

File: src/main/java/com/iridium/iridiumskyblock/configs/Placeholders.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package com.iridium.iridiumskyblock.configs;
+
+import com.iridium.iridiumcore.dependencies.fasterxml.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Placeholders {
+
+    // Island Placeholders.
+    public String island_name = "N/A";
```
Comment (dlsf) [inline src/main/java/com/iridium/iridiumskyblock/configs/Placeholders.java:9]: I'm afraid this isn't python, please use lowerCamelCase 👀

**Review by dlsf — CHANGES_REQUESTED** [review]

**Review by MertUnverdi — APPROVED** [review]
we don't need new config files but peaches is dum

**Review by dlsf — APPROVED** [review]
No, Ry is dum

**Comment by PeachesMLG** [comment]
Ry very dum

**Pull request merged**